api/test/helpers/s3.js

```diff
@@ -0,0 +1,42 @@
+const sinon = require('sinon');
```

Comment:
This file is copied 1:1 from [Bryce's branch](https://github.com/synapsestudios/oidc-platform/blob/test/storage-adapter/api/test/helpers/s3.js), hoping to avoid conflicts related to merge order.